redux/auth/saga.ts

```diff
@@ -1,8 +1,9 @@
 import * as Facebook from 'expo-facebook';
+import * as Google from 'expo-google-app-auth';
 import decode from 'jwt-decode';
 import { call, delay, put, takeLatest } from 'redux-saga/effects';
 
-import { FACEBOOK_APP_ID } from '../../env';
+import { ANDROID_CLIENT_ID, FACEBOOK_APP_ID, IOS_CLIENT_ID } from '../../env';
```

Comment:
New ENV variables are not documented yet. Please add them in our `.env.example.ts` file

Reply:
OK, changes has been pushed. I'm waiting for approval :smile: